client/java/src/test/java/org/jolokia/client/J4pClientTest.java

```diff
@@ -177,12 +178,13 @@ private HttpClient prepareMocks(String encoding,String jsonResp) throws IOExcept
         HttpEntity entity = createMock(HttpEntity.class);
         expect(client.execute(EasyMock.<HttpUriRequest>anyObject())).andReturn(response);
         expect(response.getEntity()).andReturn(entity);
+        expect(entity.isStreaming()).andReturn(false);
         expect(entity.getContentEncoding()).andReturn(encoding != null ? new BasicHeader("Content-Encoding",encoding) : null);
 
         final ByteArrayInputStream bis =
                 new ByteArrayInputStream(jsonResp.getBytes());
         expect(entity.getContent()).andReturn(bis);
-        entity.consumeContent();
+        //EntityUtils.consume(entity);
```

Comment:
Probably that's a wrong commit. I'll fix that. Yes, I'll update also the documentation. Thx for your review.